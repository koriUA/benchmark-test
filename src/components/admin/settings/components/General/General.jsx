import React from 'react';

const General = () => <div>Here goes content for General</div>;

export default General;
