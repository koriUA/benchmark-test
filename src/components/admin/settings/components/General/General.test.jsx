import React from 'react';
import { mount } from 'enzyme';

import General from './General';

describe('General', () => {
  test('should render the General', () => {
    const component = mount(<General />);
    expect(component).toMatchSnapshot();
  });
});
