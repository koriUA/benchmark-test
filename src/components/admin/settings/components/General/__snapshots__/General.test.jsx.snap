// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`General should render the General 1`] = `
<General>
  <div>
    Here goes content for General
  </div>
</General>
`;
